tests/components/blebox/test_init.py

```diff
@@ -44,7 +44,7 @@ async def test_setup_failure_on_connection(hass, caplog):
 
 async def test_unload_config_entry(hass):
     """Test that unloading works properly."""
-    patch_product_identify(None)
+    setup_product_mock("switches", [])
```

Comment:
I had to adjust the test here because `product.features` was being incorrectly patched as a coroutine.